sources/packages/backend/apps/api/src/auth/user-types.enum.ts

```diff
@@ -1,6 +1,7 @@
 export enum InstitutionUserTypes {
   admin = "admin",
   user = "user",
+  readOnlyUser = "read-only-user",
```

Comment:
For consistency, should we also update these two enums?
- https://github.com/bcgov/SIMS/blob/12a06c68c0c4cc5496f62ae3440ad119256477ca/sources/packages/backend/apps/api/src/types/institutionAuth.ts
- https://github.com/bcgov/SIMS/blob/2cfff523c1ddccffec4664f4402bd9a676a9270c/sources/packages/web/src/types/contracts/institution/InstitutionUser.ts